addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -29,6 +29,7 @@ class RecruitmentSource(models.Model):
     has_domain = fields.Char(compute='_compute_has_domain')
     job_id = fields.Many2one('hr.job', "Job", ondelete='cascade')
     alias_id = fields.Many2one('mail.alias', "Alias ID")
+    medium_id = fields.Many2one('utm.medium', "Medium ID", default=lambda self: self.env.ref('utm.utm_medium_website'))
```

Comment:
The field is already coming from the mixin
```suggestion
    medium_id = fields.Many2one(default=lambda self: self.env.ref('utm.utm_medium_website'))
```